core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -2486,10 +2486,16 @@ public FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFile
     boolean ownerGroupChanged = false;
     boolean permissionChanged = false;
     if (options.getOwner() != null) {
+      if (options.getOwner().isEmpty()) {
```

Comment:
Hmm... I don't think journal replay would call `setAttribute`, instead it directly copies the inode from the journal entry. This is replaced anyways by checking on the option